process/p2p/InterceptedShardValidatorInfo.go

```diff
@@ -0,0 +1,113 @@
+package p2p
+
```

Comment:
usually we start file names with lower case, i think it's better to change it, for consistency